lms/static/sass/features/_course-experience.scss

```diff
@@ -152,9 +156,6 @@
 
 // Course sidebar
 .course-sidebar {
-  @include margin-left(0);
-  @include padding-left($baseline);
-
   .section-tools li:not(:first-child) {
```

Comment:
Something quirky happened to the alignment of course tools and their icons 
<img width="171" alt="screen shot 2017-10-06 at 2 14 13 pm" src="https://user-images.githubusercontent.com/3861052/31292256-b6681902-aaa0-11e7-970e-2a3e070a6c0c.png">

Reply:
I didnt find the specific line / class for this but the h6 headings for .section-tools is large, and should be dropped down to match the existing course home. This change involves all sidebar headers, so its likely not scoped to just section tools but I thought I'd note it somewhere

Reply:
Nice catch. That rule lived in a generic Pattern Library partial so it didn't get included in the Bootstrap versions. I moved it into _course-experience.scss.